src/engine.cc

```diff
@@ -179,11 +178,6 @@ void EngineController::Go(const GoParams& params) {
   auto limits = PopulateSearchLimits(tree_->GetPlyCount(),
                                      tree_->IsBlackToMove(), params);
 
-  BestMoveInfo::Callback best_move_callback = [this](const BestMoveInfo& info) {
-    best_move_callback_(info);
-    GarbageCollectNodePool();
```

Comment:
Why wasn't this one working?

Reply:
Because I passed `best_move_callback_`, not`best_move_callback`.
Removing underscore would fix it, but it's even better with GC thread.